src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -87,16 +82,32 @@ angular
             y: -5
         }]
 
-    thresholdsMarkers: (options = {})->
-      return {} if _.isEmpty(options.thresholds)
-      lines = []
+    addThresholds: (options = @options)->
```

Comment:
Also, I think that you should pass an argument there to decide whether or not you want to display the threshold only in the future: we may want to display it on the full time period on other widgets